lib/findJsModulesFor.js

```diff
@@ -27,6 +27,7 @@ function findImportsFromEnvironment(
 }
 
 const PACKAGE_NAME_PATTERN = /\.\/node_modules\/([^/]+)\//;
+const SCOPED_PACKAGE_NAME_PATTERN = /\.\/node_modules\/(@[^/]+\/[^/]+)\//;
```

Comment:
Small nit, but I think the `@` part could be part of the above regexp: 

```javascript
const PACKAGE_NAME_PATTERN = /\.\/node_modules\/(?:@[^/]+\/)?([^/]+)\//;
```
(Note: this is untested and likely to not work directly, because regexps)